tests/init/config-initializer.js

```diff
@@ -8,11 +8,11 @@
 // Requirements
 //------------------------------------------------------------------------------
 
+import fsp from "node:fs/promises";
```

Comment:
We're rewriting it in [another PR](https://github.com/eslint/create-config/pull/81), and this file will be removed.

I can remove `shelljs` in that PR, thoughts?

Reply:
Sounds good 👍🏻 It looks like shelljs isn't used at all in that branch, so you could simply remove the package.json dependency.

Reply:
done in 46027138738a8296a39dfd21b0221e36d9d1f805.